### PR TITLE
RR-37 - Rotate between the users

### PR DESCRIPTION
**Pull request by nenedac**

### CODE REVIEW

File: src/app/api/v1/slack/rotation/on-duty-post/route.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    SlackResponseType,
+    sanitizeSlackText,
+    parsePayloadFromRequest,
+} from 'utils/slack';
+import {
+    filterUserOnDuty,
+    filterSlackUsers,
+    selectSlackUser
+} from 'utils/logic';
+import { SlackUser } from '@/_types';
```
Comment (marandino) [inline src/app/api/v1/slack/rotation/on-duty-post/route.ts:14]:
```suggestion
import { SlackUser } from 'types';
```

nit

File: src/app/api/v1/slack/rotation/on-duty-post/route.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+        const users = await DbClient.rotateUsers(
+            userOnDuty.slack_id,
+            newBackup,
+            previousBackup
+        );
+
+        const slackMessage = getSlackMessage(
+            SlackResponseType.InChannel,
+            `Assigned user on duty: <@${userOnDuty.slack_id}>, backup user assigned: <@${newBackup}>.`
```
Comment (marandino) [inline src/app/api/v1/slack/rotation/on-duty-post/route.ts:48]: Not really sure about this wording, it doesn't even say what the user has been assigned to, and saying "users" is quite redundant, we know they're users. How about we go with something like:
```suggestion
            `${rotationName}: <@${userOnDuty.slack_id}> is on duty, and <@${newBackup}> will back them up.`
```

rotationName is currently having underscode, so maybe a rotationName.replace("_", " ").capitalize() will be necessary.

**Review by marandino — APPROVED** [review]
lgtm, but address the feedback before merging pls

**Review by K3iiX — APPROVED** [review]
YLBV

**Pull request merged**